tests/functional/api/groups/members_test.py

```diff
@@ -62,179 +62,135 @@ def test_it_returns_empty_list_if_no_members_in_group(self, app):
 
         assert res.json == []
 
-    @staticmethod
-    def authorization_header(token) -> dict:
-        """Return an Authorization header for the given developer token."""
-        return {"Authorization": "Bearer {}".format(token.value)}
-
 
 class TestAddMember:
-    def test_it_returns_http_204_when_successful(
-        self, app, auth_client, db_session, factories
+    def test_it(self, db_session, group, user, do_request):
+        do_request()
+
+        assert user in group.members
+
+    def test_it_does_nothing_if_the_user_is_already_a_member_of_the_group(
```

Comment:
This replaces a previous test that was called [`test_it_is_idempotent()`](https://github.com/hypothesis/h/blob/1870fa8ec3386df8ae4b82820dcdbf5e517caa83/tests/functional/api/groups/members_test.py#L124-L144). I think this is a better test